package/batocera/core/batocera-audio/alsa/batocera-audio

```diff
@@ -4,8 +4,18 @@ do_usage() {
 	echo "$1 list"
 	echo "$1 get"
 	echo "$1 set {auto|<device>}"
+	echo "$1 getSystemVolume"
+	echo "$1 setSystemVolume {mute|unmute|mute-toggle|<0...150>}"
 	echo "$1 test"
-}	
+}
+
+# Export XDG for PW
+export XDG_RUNTIME_DIR=/var/run
+
+# Set variables
+MAX_VOLUME="$(/usr/bin/batocera-settings-get audio.volume.boost)" || MAX_VOLUME=100
```

Comment:
This is still not fail safe
I'm really not kidding. You are changing a script that acts systemwide. 
Consider a boolean value and nothing else

Reply:
i've not understood what is the issue and by what you would replace.

Reply:
Instead of setting a value inside batocera.conf how much the volume can be raised just enable the boost with 1 and set maximum Volume inside batocera-audio.
so 150 if the boost enabled and 100 if not enabled.

Currently you can set a value like abc and it will not work. It's not a good written script because it can obviously fail.

Reply:
```
[ $(batocera-settings-get audio.volume.boost) -eq 1 ] && MAX_VOLUME=150 || MAX_VOLUME=100
```
Would solve all issues

Reply:
Except it is wanted that user can raise volume without limit. But @liberodark seems to be quiet about this or does not know how fragile his scripting is.

Reply:
I added a value control.
On the PR.
In my opinion this is sufficient.
I leave this point to susan.